aws-datastore/src/main/java/com/amplifyframework/datastore/appsync/ModelMetadata.java

```diff
@@ -37,7 +38,7 @@
     private final @ModelField(targetType = "ID", isRequired = true) String id;
     private final @ModelField(targetType = "Boolean") Boolean _deleted;
     private final @ModelField(targetType = "Int") Integer _version;
-    private final @ModelField(targetType = "AWSTimestamp") Long _lastChangedAt;
```

Comment:
Will this require the SQL table to be reprovisioned?

Reply:
^If so, will this be considered a breaking change?

Reply:
No, it will not require the SQL table to be reprovisioned, since it will still be stored as a `Long` either way.  That said, thanks for asking this question - I realized that the SQLiteModelFieldTypeConverter doesn't support `AWSTimestamp` yet, so I will add that.